### PR TITLE
Replace iron doc viewer with something that looks a lot like it.

### PR DESCRIPTION
**Pull request by andymutton**

### CODE REVIEW

File: client/src/catalog-api-property.html [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+        /**
+         * Resets any state that was set up for transitions.
+         *
+         * We are careful to reset our explicit heights after a transition
+         * completes, so that the property doesn't clip values if the user resizes
+         * their viewport.
+         * @param {object} event The event in question.
+         */
+        _onTransitionEnd: function(event) {
```
Comment (dstockwell) [inline client/src/catalog-api-property.html]: A bunch of these functions don't seem to be used.

File: client/src/catalog-api-property.html [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+        display: none;
+      }
+
+      .deeplink {
+        color: currentcolor;
+        text-decoration: none;
+      }
+
+      .deeplink:hover {
+        color: var(--paper-pink-500);
```
Comment (dstockwell) [inline client/src/catalog-api-property.html]: We should try to avoid paper colors.

File: client/src/catalog-api-property.html (hunk 1 above)
Comment (andymutton) [inline client/src/catalog-api-property.html]: I've deleted a bunch of functions.

File: client/src/catalog-api-property.html (hunk 2 above)
Comment (andymutton) [inline client/src/catalog-api-property.html]: Doned it.

**Pull request merged**